app/src/main/java/net/sourceforge/opencamera/MainActivity.java

```diff
@@ -1772,13 +1792,50 @@ public void clickedStamp(View view) {
         preview.showToast(stamp_toast, value ? R.string.stamp_enabled : R.string.stamp_disabled);
     }
 
-    public void clickedSwitchRecSync() {
+    @RequiresApi(api = Build.VERSION_CODES.N)
+    public void clickedEnableRecSync() {
         if( MyDebug.LOG )
-            Log.d(TAG, "clickedSwitchRecSync");
+            Log.d(TAG, "clickedEnableRecSync");
+
+        if( applicationInterface.getEnableRecSyncPref() ) {
+            applicationInterface.startSoftwareSync();
+        } else {
+            applicationInterface.stopSoftwareSync();
+        }
 
-        // TODO show client/leader info
-        // TODO open video mode
-        // in MyApplicationInterface?
+        applicationInterface.getDrawPreview().updateSettings(); // because we cache the enable RecSync setting
+    }
+
+    public void clickedSyncSettings() {
+        if( MyDebug.LOG )
```

Comment:
If we do this, the whole original file will be changed, because the original codestyle does not meet IDEA's requirements. We thought it is better to just follow the original rules.